consul/files/template.json.jinja

```diff
@@ -0,0 +1 @@
+{{ content | tojson(indent=2) }}
```

Comment:
The `tojson()` Jinja filter is only available in the latest stable Salt 2018.3.3 release.
However, the [built-in serializer](https://docs.saltstack.com/en/latest/ref/renderers/all/salt.renderers.jinja.html) has generic JSON filter for ages and also supports indentation. Simply like this:
```
content|json(indent=2)
```

This is more portable and reliable. As far as I understand upstream Jinja docs, the `tojson()` filter is designed to be used within HTML pages, and it could do some weird things when you need "raw" JSON file.